app/controllers/api/v1/accounts/following_accounts_controller.rb

```diff
@@ -6,6 +6,7 @@ class Api::V1::Accounts::FollowingAccountsController < Api::BaseController
   after_action :insert_pagination_headers
 
   def index
+    cache_if_unauthenticated!
```

Comment:
This might increase cases of “inconsistent” pages, i.e. someone unfollows and refollows and is listed across 2 pages.

We may want to improve the WebUI handling first (because as it stands, I think the WebUI will include duplicate components with duplicate keys, leading into various issues).